### PR TITLE
Use new pagination component on search pages

### PR DESCRIPTION
**Pull request by domoscargin**
https://trello.com/c/d1wSRrnT/179-2-use-govuk-style-for-previous-and-next-links-on-search-results-page

### CODE REVIEW

**Comment by lfdebrux** [comment]
### My review checklist

I've:

- [ ] looked at the code
- [ ] tested locally
- [ ] run functional tests locally

File: app/templates/search/_pagination.html [hunk 1]
```diff
@@ -1,30 +1,24 @@
-<nav role="navigation">
-    <ul class="previous-next-navigation">
+{% if not dmPreviousNextPagination %}
+  {% from "digitalmarketplace/components/previous-next-pagination/macro.njk" import dmPreviousNextPagination %}
+{% endif %}
```
Comment (lfdebrux) [inline app/templates/search/_pagination.html:3]: I'm not sure the import guard is necessary

Comment (domoscargin) [reply app/templates/search/_pagination.html:3]: I thought so, but it ends up failing this test with a `'?live-results=true'` arg:

https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/37f0d7da87b41843f809c107b7cd376d6487308f/tests/main/views/test_marketplace.py#L1794-L1800

I think it's related to this bit here which seems to only be rendering the parts it needs, so it doesn't catch the base page?
https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/37f0d7da87b41843f809c107b7cd376d6487308f/app/main/views/marketplace.py#L349-L373

Comment (lfdebrux) [reply app/templates/search/_pagination.html:3]: Ah, alright then. Thanks for explaining that.

**Review by lfdebrux — APPROVED** [review]
Looks good to me 👍

**Pull request merged**